source/index.js

```diff
@@ -1,22 +1,14 @@
 require("dotenv").config();
 
 const NAME = "{{service.name}}";
+const definitions = require("./definitions");
```

Comment:
i would rather to not import definition, let it be an empty array in the boilerplate, or an array with one sample resource